loki_simple.md

```diff
@@ -0,0 +1,252 @@
+# Automatic deployment Loki with the Network Observability Operator
+
+In its current status, deploying and configuring the Loki operator with our NOO
+[might be a complex task](./hack_loki.md). Basically due to the setup and configuration
+of the gateway and the external dependencies towards the Storage Engine. In addition,
+different Cloud providers might require different steps and configuration.
```

Comment:
Yes, but my understanding is that this specificities are optional and we could expose them with the loki section of the flowcollector.

Reply:
I thought gateway was optional but not the S3 information (credentials, chunk ID...).